update.sh

```diff
@@ -1,83 +1,88 @@
-#!/bin/sh
+#!/bin/bash
+set -xeo pipefail
 
 ARCHS=${ARCHS:-"alpha amd64 arm64 armel armhf hppa hurd-i386 i386 kfreebsd-amd64 kfreebsd-i386 m68k mips mipsel powerpc powerpcspe ppc64 ppc64el s390x sh4 sparc64 x32"}
 
-set -e
+wget_opts="-t 3 -w 1 --retry-connrefused --no-dns-cache --retry-on-http-error=403"
 
 for arch in ${ARCHS}; do
     mkdir -p "${arch}/slim"
 
     # arch
     if [ ! -f "${arch}/arch" ]; then
-	echo "${arch}" > "${arch}/arch"
+        echo "${arch}" > "${arch}/arch"
     fi
     qemu_arch=$(cat "${arch}/qemu_arch")
 
     # deb
     if [ ! -f "${arch}/deb" ]; then
-	(
-	    set -x
-	    curl -s https://packages.debian.org/sid/${arch}/busybox-static/download | tr '"' "\n" | grep -E "http://.*${arch}\\.deb" > "${arch}/deb"
-	)
+        (
+            set -x
+            curl -s https://packages.debian.org/sid/${arch}/busybox-static/download | tr '"' "\n" | grep -E "http://.*${arch}\\.deb" > "${arch}/deb"
+        )
     fi
     deb=$(head -n 1 "${arch}/deb")
 
     if [ ! -f "${arch}/busybox-static.deb" ]; then
-	(
-	    set -x
-	    curl "${deb}" > "${arch}/busybox-static.deb.tmp"
-	    mv "${arch}/busybox-static.deb.tmp" "${arch}/busybox-static.deb"
-	)
+        (
+            set -x
+            wget ${wget_opts} "${deb}" -O "${arch}/busybox-static.deb"
+        )
     fi
 
     # extract /bin/busybox
     if [ ! -f "${arch}/slim/rootfs/bin/busybox" ]; then
-	(
-	    cd "${arch}"
-	    set -x
-	    ar vx busybox-static.deb data.tar.xz data.tar.gz
-	    tar --strip=2 -xvf data.tar.* ./bin/busybox
-	    mkdir -p ./slim/rootfs/bin
-	    mv ./busybox ./slim/rootfs/bin/
-	)
+        pushd "${arch}"
+        ar vx busybox-static.deb data.tar.xz data.tar.gz
+        tar --strip=2 -xvf data.tar.* ./bin/busybox
+        mkdir -p ./slim/rootfs/bin
+        mv ./busybox ./slim/rootfs/bin/
+        ls -l $(pwd)/slim/rootfs/bin/busybox
+        popd
     fi
 
     # create symlinks
     if [ ! -f "${arch}/slim/rootfs/bin/ls" ]; then
-	(
-	    cd "${arch}/slim/rootfs"
-	    echo "This need binfmt to be configured"
-	    echo "  docker run --rm --privileged multiarch/qemu-user-static:register --reset."
-	    for module in $("./bin/busybox" --list-modules); do
-		mkdir -p "$(dirname $module)"
-		ln -s /bin/busybox "${module}"
-	    done
-	)
+        pushd "${arch}/slim/rootfs"
+        echo "This need binfmt to be configured"
+        echo "  docker run --rm --privileged multiarch/qemu-user-static:register --reset."
+        ls -l $(pwd)/bin
+        ls -l $(pwd)/bin/busybox
+        $(pwd)/bin/busybox --help
+        $(pwd)/bin/busybox --list-modules
```

Comment:
There is an interesting error here. `$(pwd)/bin/busybox --help` is not recognized

https://travis-ci.org/multiarch/busybox/builds/543095257

```
++pwd
+ls -l /home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox
-rwxr-xr-x 1 travis travis 2382376 Apr  1 05:17 /home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox
++pwd
+/home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox --help
./update.sh: line 51: /home/travis/build/multiarch/busybox/alpha/slim/rootfs/bin/busybox: No such file or directory
```